frontend/src/components/Layout.tsx

```diff
@@ -0,0 +1,28 @@
+import { FC } from 'react';
+
+import { APPLICATION_TITLE } from '../constants/labels';
+import { CASTLE_SOURCE, SETTINGS_ICON_SOURCE, TILE_TITLE_SOURCE } from '../constants/layoutElements';
+
+export const Header: FC = () => (
```

Comment:
Let's change the name to AppHeaderSection or AppHeaderContainer. Header is too general